pygfx/renderers/wgpu/wgsl/points.wgsl

```diff
@@ -464,6 +465,9 @@ fn get_signed_distance_to_shape_edge(coord_in: vec2<f32>, size: f32) -> f32 {
         let r4 = length(coord-c1)-size/5;
         return max( min(r1,max(max(r2,r3),-coord.y)), -r4);
 
+    $$ elif shape == 'custom'
+        {{ custom_sdf }}
```

Comment:
I think it would be good to put all of this in a clean function that doesn't have access to all the niceties of variables defined in `get_signed_distance_to_shape_edge`.

Reply:
it could be that the signature is simply that of `get_signed_distance_to_shape_edge`

Reply:
done in https://github.com/pygfx/pygfx/pull/822

Reply:
feel free to resolve these discussion threads of you feel they are "done". they'll only block the PR and perhaps unintentionally create the impression you're still working on it.